lib/build.gradle.kts

```diff
@@ -96,7 +96,18 @@ dependencies {
     coreLibraryDesugaring(libs.android.desugaring)
 
     implementation(libs.androidx.core)
-    api(libs.ical4j)
+    api(libs.ical4j) {
+        // Get rid of unnecessary transitive dependencies
+        exclude(group = "commons-validator", module = "commons-validator")
+        exclude(group = "org.slf4j", module = "slf4j-api")
```

Comment:
We include `org.slf4j:slf4j-jdk14` later ourself, which depends on `slf4j-api`. So we don't need to exclude it here. ical4j does logging over sl4fj-api; I think it would break without (if we wouldn't import it ourselves below).